packages/core/storybook/Stamp.js

```diff
@@ -12,34 +12,56 @@ storiesOf('Stamp', module)
         'Use the <Stamp /> component to subtly display attributes alongside listing cells and on product detail pages. Use it in conjunction with an <Icon /> component to give it more context. An Icon placed within a Stamp will inherit the assigned Stamp color.'
     })(() => <Stamp>default stamp</Stamp>)
   )
+  .add('Small Stamp', () => (
+    <Stamp color="primary" variation="small">
+      <Icon name="Pin" mr={1} />
+      Stamp Text
+    </Stamp>
+  ))
+  .add('Regular Stamp', () => (
+    <Stamp color="primary">
+      <Icon name="Pin" mr={1} />
+      Stamp Text
+    </Stamp>
+  ))
   .add('All Colors', () => (
     <div>
-      <Stamp color="gray" mr={2}>
+      <Stamp color="text.light" mr={2}>
```

Comment:
Thanks!

Reply:
I looking at creating a similar to variations util to handle this instead since we want a solid small variation as well

Reply:
https://priceline.invisionapp.com/share/R2WMAJ03ABU#/screens